codalab/lib/bundle_cli.py

```diff
@@ -1192,6 +1208,119 @@ def do_run_command(self, args):
         self.wait(client, args, new_bundle['uuid'])
 
     @Commands.command(
+        'run-local',
+        help='Run a run bundle locally.',
+        arguments=(
+            Commands.Argument('target_spec', help=ALIASED_TARGET_SPEC_FORMAT, nargs='*', completer=TargetsCompleter),
+            Commands.Argument('command', metavar='[---] command', help='Arbitrary Linux command to execute.', completer=NullCompleter),
+            Commands.Argument('-w', '--worksheet-spec', help='Operate on this worksheet (%s).' % WORKSHEET_SPEC_FORMAT, completer=WorksheetsCompleter),
+        ) + Commands.metadata_arguments([RunBundle]) + EDIT_ARGUMENTS + WAIT_ARGUMENTS,
+    )
+    def do_run_local_command(self, args):
+        client, worksheet_uuid = self.parse_client_worksheet_uuid(args.worksheet_spec)
+        args.target_spec, args.command = cli_util.desugar_command(args.target_spec, args.command)
+        targets = self.parse_key_targets(client, worksheet_uuid, args.target_spec)
+        metadata = self.get_missing_metadata(RunBundle, args)
+
+        docker_image = metadata.get('request_docker_image', 'ubuntu:14.04')
+        if not docker_image:
+            docker_image = 'ubuntu:14.04'
+
+        uuid = generate_uuid()
+        bundle_path = '/opt/{}'.format(uuid)
+        command = args.command
+        request_network = None
+        dependencies = [
+            (u'{}'.format(key), u'/{}_dependencies/{}'.format(uuid, key)) for key, target in self.parse_target_specs(args.target_spec)
+        ]
+
+        # Set up a directory to store the bundle.
+        remove_path(bundle_path)
+        os.mkdir(bundle_path)
+
+        for path1, path2 in dependencies:
```

Comment:
use `name` and `child_path` (I think) to maintain consistency with code elsewhere.